BMS Workspace-git/bms-1227-fw/targetConfigs/TMS570LS1224.ccxml

```diff
@@ -1,14 +1,14 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<configurations XML_version="1.2" id="configurations_0">
-<configuration XML_version="1.2" id="SEGGER J-Link Emulator_0">
-        <instance XML_version="1.2" desc="SEGGER J-Link Emulator_0" href="connections/segger_j-link_connection.xml" id="SEGGER J-Link Emulator_0" xml="segger_j-link_connection.xml" xmlpath="connections"/>
-        <connection XML_version="1.2" id="SEGGER J-Link Emulator_0">
-            <instance XML_version="1.2" href="drivers/JLINKicepick_c.xml" id="drivers" xml="JLINKicepick_c.xml" xmlpath="drivers"/>
-            <instance XML_version="1.2" href="drivers/jlinkcs_dap.xml" id="drivers" xml="jlinkcs_dap.xml" xmlpath="drivers"/>
-            <instance XML_version="1.2" href="drivers/jlinkcortexr4.xml" id="drivers" xml="jlinkcortexr4.xml" xmlpath="drivers"/>
-            <platform XML_version="1.2" id="platform_0">
-                <instance XML_version="1.2" desc="TMS570LS1224_0" href="devices/tms570ls1224.xml" id="TMS570LS1224_0" xml="tms570ls1224.xml" xmlpath="devices"/>
-            </platform>
-        </connection>
-    </configuration>
-</configurations>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<configurations XML_version="1.2" id="configurations_0">
+<configuration XML_version="1.2" id="Texas Instruments XDS110 USB Debug Probe_0">
+        <instance XML_version="1.2" desc="Texas Instruments XDS110 USB Debug Probe_0" href="connections/TIXDS110_Connection.xml" id="Texas Instruments XDS110 USB Debug Probe_0" xml="TIXDS110_Connection.xml" xmlpath="connections"/>
+        <connection XML_version="1.2" id="Texas Instruments XDS110 USB Debug Probe_0">
+            <instance XML_version="1.2" href="drivers/tixds510icepick_c.xml" id="drivers" xml="tixds510icepick_c.xml" xmlpath="drivers"/>
+            <instance XML_version="1.2" href="drivers/tixds510cs_dap.xml" id="drivers" xml="tixds510cs_dap.xml" xmlpath="drivers"/>
+            <instance XML_version="1.2" href="drivers/tixds510cortexR.xml" id="drivers" xml="tixds510cortexR.xml" xmlpath="drivers"/>
+            <platform XML_version="1.2" id="platform_0">
+                <instance XML_version="1.2" desc="TMS570LS1224_0" href="devices/tms570ls1224.xml" id="TMS570LS1224_0" xml="tms570ls1224.xml" xmlpath="devices"/>
+            </platform>
+        </connection>
+    </configuration>
+</configurations>

```

Comment:
We should be able to flash the launchpad code with no problem now. Thanks for fixing this!